src/Microsoft.IdentityModel.Protocols/Configuration/StaticConfigurationManager.cs

```diff
@@ -44,7 +44,7 @@ public class StaticConfigurationManager<T> : IConfigurationManager<T> where T :
         /// <summary>
         /// Initializes an new instance of <see cref="StaticConfigurationManager{T}"/> with a Configuration instance.
         /// </summary>
-        /// <param name="configuration">Configuration of type <see cref="OpenIdConnectConfiguration"/> or <see cref="WsFederationConfiguration"/>.</param>
+        /// <param name="configuration">Configuration of type OpenIdConnectConfiguration or OpenIdConnectConfiguration.</param>
```

Comment:
The `OpenIdConnectConfiguration` exists in `Microsoft.IdentityModel.Protocols.OpenIdConnect` namespace where the above class `StaticConfigurationManager` exists in `Microsoft.OpenIdConnect.Protocols` namespace. OIDC depends on `Microsoft.OpenIdConnect.Protocols`, how can we refer to `OpenIdConfiguration`. When I try to do that, intellisense does not pull up the class name.